core/src/test/java/apoc/meta/MetaTest.java

```diff
@@ -1539,4 +1540,36 @@ public void testMetaNodesCount() {
         
     }
 
+    @Test
+    public void testRelTypePropertiesMovies() throws Exception {
+        final String query = IOUtils.toString(new InputStreamReader(Thread.currentThread().getContextClassLoader().getResourceAsStream("movies.cypher")));
+
+        db.executeTransactionally(query);
+        
+        TestUtil.testResult( db, "CALL apoc.meta.relTypeProperties($config)",
+                Map.of("config", Map.of("includeRels", List.of("REVIEWED"))), r -> {
+                    // System.out.println("r.resultAsString() = " + r.resultAsString());
```

Comment:
I think you should clean up this print line

Reply:
done